srv/salt/_modules/rgw.py

```diff
@@ -104,9 +107,19 @@ def add_users(pathname="/srv/salt/ceph/rgw/cache"):
                 log.info("stderr: {}".format(line))
 
             proc.wait()
-        
-        
-        
+
+
+def create_bucket(**kwargs):
+    s3conn = boto.connect_s3(
+        aws_access_key_id=kwargs['access_key'],
+        aws_secret_access_key=kwargs['secret_key'],
+        host=kwargs['host'],
+        is_secure=False,
```

Comment:
can you get the is_secure value from the way we derive rgw ui credentials, if the radosgw is https terminated setting is_secure as false would fail